iotdb-protocol/thrift-commons/src/main/thrift/common.thrift

```diff
@@ -90,12 +90,28 @@ struct TDataNodeConfiguration {
   2: required TNodeResource resource
 }
 
+// TODO: deprecated
 enum TRegionMigrateFailedType {
```

Comment:
remove this field

Reply:
For some reason it seems still useful, maybe remove it in next version ?